pepper-apis/studybuilder-cli/studies/osteo/emails/self_consent_completed_es.html

```diff
@@ -0,0 +1,118 @@
+        <img src="https://storage.googleapis.com/${assetsBucketName}/osteo/project-logo.png"
+             alt="Osteosarcoma Project logo" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;"
+             width="230px" height="63px" border="0">
+      </a>
+    </td>
+  </tr>
+
+  <tr style="width: 540px;">
+    <td style="padding: 50px 0 0 0;">
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Estimado(a) {{participant_firstName}},</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Gracias por unirse al Proyecto sobre Osteosarcoma y por darnos su consentimiento para la investigación. Su participación no solo es importante para nuestro trabajo, sino que impulsa todo lo que hacemos.</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">A continuación, le enviaremos por correo postal un kit con instrucciones sobre cómo proporcionar una muestra de saliva. También podríamos ponernos en contacto con usted con relación a proporcionar una muestra de sangre (un tubo o dos cucharaditas), que puede extraerse en el consultorio de su médico, su clínica local o un laboratorio cercano. Le daremos instrucciones detalladas sobre cómo hacerlo. Es posible que obtengamos copias de su historia clínica y, después de revisarla, también podríamos solicitar una porción de la muestra del tumor almacenada al hospital donde se encuentra. Una vez que hayamos analizado sus muestras, devolveremos las muestras del tumor que no se hayan utilizado al departamento de anatomía patológica que nos las haya enviado. En este momento, solo podemos recolectar muestras de personas que viven en los EE. UU. y Canadá. Sin embargo, esperamos incluir pronto a pacientes internacionales.</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">i aún no ha proporcionado su dirección postal ni la información de contacto de sus médicos y hospitales, puede completar el proceso aquí:</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="{{baseWebUrl}}/activity-link/{{activityInstanceGuid}}?participantGuid=-ddp.participant.guid-" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```

Comment:
This should be dynamic convention {{participantGuid}}
Please search for -ddp in *_es.html files for such stuff missed

Reply:
@aweng98 ^^ looks like this is missed
Please search for -ddp in ALL  *_es.html files for such stuff missed from your IDE

Reply:
hmm, can't believe i missed that. Just checking, if pattern was not changed for some variables like this, it is not going to break the app, right? Should we update them in all es *.html too?

Reply:
I search for `-ddp.` in `emails` dir, I did not find any occurrence in any `*_es.html` files. It should be the only one I had missed.